assets/scripts/auth/api-user.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const app = require('../app-data');
+
+const signUp = (success, failure, data) => {
+  $.ajax({
+    method: "POST",
+    url: app.api + '/sign-up',
+    data,
+
+  })
+  .done(success)
```

Comment:
Made the changes as shown in 
https://github.com/PizzaBeer/filebucket-front-end/commit/3cf0c94b7441802b6242efe5aacc81e675f2cfc0

How does that look?